src/Troubleshooting/index.md

```diff
@@ -95,77 +54,34 @@ There's no known fix (as of October 14th, 2023), but the following workarounds m
   If none of the workarounds above work or apply, consider using Roblox windowed instead of fullscreen.
   This may also be your GPU not supporting Vulkan or not supporting modern Vulkan; Use the OpenGL renderer, or set the installed DXVK version to one which includes a legacy version of Vulkan that your GPU supports.
 
-### UI elements look small
```

Comment:
The removal of this section should be reverted, since some users have other scale values.